x-pack/legacy/plugins/canvas/i18n/ui.ts

```diff
@@ -308,6 +308,7 @@ export const ArgumentStrings = {
 };
 
 export const DataSourceStrings = {
+  // Demo data source
```

Comment:
Do you feel like these comments are helpful? For me, I feel like the object key being `DemoData` does a good enough job of describing what's going on and the comment is just noise, but if the comment is helpful, then keep it.

Reply:
I found them helpful, as it was not intuitive to me going through the code, and some datasources were in different files. `DemoData` was intuitive, but `ESDocs` wasn't, so I just added comments for all for consistency. Also, with the i18n {} values, it was difficult to search and find the file I needed without the comments.